meta-ostro/lib/oeqa/selftest/iotsstatetests.py

```diff
@@ -127,7 +127,7 @@ def get_hashes(d, subdir):
                                      stdout=subprocess.PIPE,
                                      stderr=subprocess.STDOUT)
                 stdout, stderr = p.communicate()
-                analysis.append(stdout)
+                analysis.append(stdout.decode("ascii"))
```

Comment:
I prefer decode("utf-8") - a bit more future-proof, but shouldn't matter in practice.